recipe/meta.yaml

```diff
@@ -11,38 +11,39 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
 
 requirements:
   host:
-    - python >=2.7
+    - python
     - pip
-    - flit-core >=3.7,<4
+    - flit-core
   run:
-    - python >=2.7
+    - python
     - sphinx >=1.8
 
 test:
   imports:
     - sphinxcontrib.jquery
-  commands:
-    - pip check
   requires:
     - pip
+  commands:
+    - pip check
 
 about:
   home: https://pypi.org/project/sphinxcontrib-jquery/
   license: 0BSD AND MIT
   license_file:
     - LICENCE
     - vendor-licenses/
```

Comment:
I don't think this points to the jQuery license. From what I see in the sdist (https://inspector.pypi.io/project/sphinxcontrib-jquery/4.1/packages/de/f3/aa67467e051df70a6330fe7770894b3e4f09436dea6881ae0b4f3d87cad8/sphinxcontrib-jquery-4.1.tar.gz/), that folder doesn't exist.

jQuery is licensed under MIT and so requires us to bundle the license. See https://github.com/jquery/jquery/blob/main/LICENSE.txt.